packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -696,13 +730,302 @@ describe("Garbage Collection Tests", () => {
 
 			summarizerContainerTests(
 				sweepTimeoutMs,
+				"sweep",
 				"GarbageCollector:SweepReadyObject_Revived",
 				"GarbageCollector:SweepReadyObject_Changed",
 				"GarbageCollector:SweepReadyObject_Loaded",
 				false, // expectDeleteLogs
 			);
 		});
 
+		describe("GC version changes", () => {
+			type GcWithStateOverride = IGarbageCollector & {
+				readonly baseSnapshotDataP: Promise<IGarbageCollectionSnapshotData | undefined>;
+				readonly tombstones: string[];
+				readonly deletedNodes: Set<string>;
+				readonly summaryStateTracker: Omit<GCSummaryStateTracker, "latestSummaryData"> & {
+					latestSummaryData: IGCSummaryTrackingData | undefined;
+				};
+			};
+
+			const createGCOverride = (
+				baseSnapshot: ISnapshotTree,
+				gcBlobsMap: Map<
+					string,
+					IGarbageCollectionState | IGarbageCollectionDetailsBase | string[]
+				>,
+				gcFeature: GCVersion,
+			) => {
+				const gcMetadata: IGCMetadata = {
+					gcFeature,
+				};
+				return createGarbageCollector(
+					{ baseSnapshot },
+					gcBlobsMap,
+					gcMetadata,
+				) as GcWithStateOverride;
+			};
+
+			function getSnapshotWithGCVersion(gcVersion: GCVersion) {
+				// Create a snapshot tree to be used as the GC snapshot tree.
+				const gcSnapshotTree = getDummySnapshotTree();
+				const gcBlobId = "root";
+				// Add a GC blob with key that start with `gcBlobPrefix` to the GC snapshot tree. The blob Id for this
```

Comment:
```suggestion
				// Add a GC blob with key that start with `blobs` to the GC snapshot tree. The blob Id for this
```

`gcBlobPrefix` is the variable name not the actual value, right?

Reply:
yeah, its not. will update.